packages/core/src/util.ts

```diff
@@ -7,7 +8,9 @@ export function loadImage(path: string) {
 }
 
 export function generateImageID(url: URL, config: ImageConfig) {
-  const baseURL = url.host ? new URL(url.origin + url.pathname) : new URL(url.protocol + url.pathname)
+  const baseURL = url.host
+    ? new URL(url.origin + url.pathname)
+    : new URL(url.protocol + path.relative(process.cwd(), url.pathname))
```

Comment:
```suggestion
    // use a relative path here so that the path is the same locally and on CI
    : new URL(url.protocol + path.relative(process.cwd(), url.pathname))
```